src/runtime_src/core/common/api/xrt_device.cpp

```diff
@@ -91,23 +93,31 @@ namespace xrt {
 
 device::
 device(unsigned int index)
-  : handle(xrt_core::get_userpf_device(index))
+    : handle(xrt_core::get_userpf_device(index))
```

Comment:
Curious

Reply:
I'm trying out CLion based on the most recent SW developer forum, and it has a code reformat utility. I still haven't found the settings which jive most perfectly with XRT, but I'm close.

Reply:
Recommend emacs :-)

Reply:
I recommend a modern and well configured Emacs too, for example with RTags or Language Server Protocol support.
But I am unsure it is possible at Xilinx by default... Recently I have see a Xilinx colleague using XEmacs, an abandoned fork with last release in 2009. I am just crying. :-(
So between the latest CLion and a Jurassic unconfigured version of Emacs, CLion might be preferable...